src/App.js

```diff
@@ -1,19 +1,7 @@
 import './App.css';
 
-import React, { Component } from 'react';
-
 import Calculator from './components/calculator';
 
-/* eslint-disable react/prefer-stateless-function */
-
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
-      </div>
-    );
-  }
-}
+const App = () => (<div className="App"><Calculator /></div>);
```

Comment:
### 1. According to the general Javascript best practices requirements:

![image](https://user-images.githubusercontent.com/57421823/153396161-8833678d-669c-4f12-904b-4b32a153124d.png)

Therefore:

- [ ] The file `App.test.js` is full of old commented code.  Please, kindly remove this commented code, or delete the file since it's not needed.

![image](https://user-images.githubusercontent.com/57421823/153394397-e22e0475-73a9-42c6-ac12-e860b86a8f09.png)


That way, the project will be cleaner.

### 2. According to the general project documentation requirements:

![image](https://user-images.githubusercontent.com/57421823/153395861-2457f05a-f223-4b5f-96e3-4aa7ff1df982.png)

Therefore:

- [ ] Please, kindly update your readme file and use this [Microverse template](https://github.com/microverseinc/readme-template).  _Be sure to include ALL sections except for the live demo and the tests when those are not required._

That way, your app will be well documented 💪 

### 3. Optional suggestions: 

- [ ] OPTIONAL: I highly suggest you remove all the styles coming from the CRA on `index.css`.

![image](https://user-images.githubusercontent.com/57421823/153395627-59cc8b8a-fc7c-45f4-b794-7cfcc9d06317.png)